app.R

```diff
@@ -18,30 +22,62 @@ ui <- fluidPage(
     # Sidebar with a slider input for number of bins
     sidebarLayout(
         sidebarPanel(
+            checkboxGroupInput(inputId = "stems_plot",
+                               label = "Census year:",
+                               choices = c("2008-2010" = "1",
+                                 "2013" = "2",
+                                 "2018" = "3")
+            ),
             sliderInput("bins",
                         "Number of bins:",
                         min = 1,
                         max = 50,
-                        value = 30)
-        ),
+                        value = 20)
+            ),
 
         # Show a plot of the generated distribution
         mainPanel(
-           plotOutput("distPlot")
-        )
+           plotOutput("histPlot")
+        ))
     )
-)
+
 
 # Define server logic required to draw a histogram
 server <- function(input, output) {
 
-    output$distPlot <- renderPlot({
+    stem1 <- here::here("data/scbi.stem1.csv") %>%
+        read_csv()
+    stem2 <- here::here("data/scbi.stem2.csv") %>%
+        read_csv()
+    stem3 <- here::here("data/scbi.stem3.csv") %>%
+        read_csv()
+
+    stems <- rbind(stem1, stem2, stem3)
+
+
+
+
+    output$histPlot <- renderPlot({
         # generate bins based on input$bins from ui.R
-        x    <- faithful[, 2]
-        bins <- seq(min(x), max(x), length.out = input$bins + 1)
+        x <- stems_plot[, 2]
```

Comment:
I think this line right here is causing the problem
<img width="704" alt="Screen Shot 2020-10-05 at 11 10 15 AM" src="https://user-images.githubusercontent.com/4574203/95097326-5a829f80-06fb-11eb-83b0-65963261ef47.png">

Reply:
Hi Albert!  So odd, it seems to be working for me.  You think it's line 62 that's the issue?  Here is what I have on my end 
<img width="1236" alt="Screen Shot 2020-10-05 at 11 17 15" src="https://user-images.githubusercontent.com/47457317/95098171-2ef3bb00-06e3-11eb-8299-6140f5bb076b.png">

Reply:
Weird, could you update all packages, then run this:
``` r
library(shiny)
library(tidyverse)
#> Warning: package 'tidyr' was built under R version 4.0.2
#> Warning: package 'dplyr' was built under R version 4.0.2
library(readr)
library(hrbrthemes)
#> Warning: package 'hrbrthemes' was built under R version 4.0.2
#> NOTE: Either Arial Narrow or Roboto Condensed fonts are required to use these themes.
#>       Please use hrbrthemes::import_roboto_condensed() to install Roboto Condensed and
#>       if Arial Narrow is not on your system, please see https://bit.ly/arialnarrow
sessionInfo()
#> R version 4.0.1 (2020-06-06)
#> Platform: x86_64-apple-darwin17.0 (64-bit)
#> Running under: macOS Catalina 10.15.6
#> 
#> Matrix products: default
#> BLAS:   /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRblas.dylib
#> LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib
#> 
#> locale:
#> [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
#> 
#> attached base packages:
#> [1] stats     graphics  grDevices utils     datasets  methods   base     
#> 
#> other attached packages:
#>  [1] hrbrthemes_0.8.0 forcats_0.5.0    stringr_1.4.0    dplyr_1.0.2     
#>  [5] purrr_0.3.4      readr_1.3.1      tidyr_1.1.2      tibble_3.0.3    
#>  [9] ggplot2_3.3.2    tidyverse_1.3.0  shiny_1.5.0     
#> 
#> loaded via a namespace (and not attached):
#>  [1] Rcpp_1.0.5        lubridate_1.7.9   assertthat_0.2.1  digest_0.6.25    
#>  [5] mime_0.9          R6_2.4.1          cellranger_1.1.0  backports_1.1.10 
#>  [9] reprex_0.3.0      evaluate_0.14     httr_1.4.2        highr_0.8        
#> [13] pillar_1.4.6      gdtools_0.2.2     rlang_0.4.7       readxl_1.3.1     
#> [17] extrafontdb_1.0   blob_1.2.1        rmarkdown_2.3     extrafont_0.17   
#> [21] munsell_0.5.0     broom_0.7.0       compiler_4.0.1    httpuv_1.5.4     
#> [25] modelr_0.1.8      xfun_0.17         systemfonts_0.3.1 pkgconfig_2.0.3  
#> [29] htmltools_0.5.0   tidyselect_1.1.0  fansi_0.4.1       crayon_1.3.4     
#> [33] dbplyr_1.4.4      withr_2.3.0       later_1.1.0.1     grid_4.0.1       
#> [37] Rttf2pt1_1.3.8    jsonlite_1.7.1    xtable_1.8-4      gtable_0.3.0     
#> [41] lifecycle_0.2.0   DBI_1.1.0         magrittr_1.5      scales_1.1.1     
#> [45] cli_2.0.2         stringi_1.5.3     fs_1.5.0          promises_1.1.1   
#> [49] xml2_1.3.2        ellipsis_0.3.1    generics_0.0.2    vctrs_0.3.4      
#> [53] tools_4.0.1       glue_1.4.2        hms_0.5.3         fastmap_1.0.1    
#> [57] yaml_2.2.1        colorspace_1.4-1  rvest_0.3.6       cpp11_0.2.2      
#> [61] knitr_1.30        haven_2.3.1
```

<sup>Created on 2020-10-05 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

For all intents and purposes you're done, but I'd still like to know why it's work for you but not me

Reply:
Also you don't have any lingering uncommitted changes?

Reply:
I just updated packages, then ran the stuff above and got this:
<img width="670" alt="Screen Shot 2020-10-05 at 20 00 23" src="https://user-images.githubusercontent.com/47457317/95144074-92eea180-072c-11eb-9d8f-686e25930706.png">

I also re-pushed changes so hopefully that fixes things!

Reply:
I think I know what the issue is. please watch the following loom screencast:
https://www.loom.com/share/59757c9524744823a4cc85f36a235657

Reply:
hey @rudeboybert I just cleared the environment, switched around the order in which I call and define stems_plot and pushed my changes!  It works on my end, but let me know if it's working for you!